docker/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM golang:1.19-alpine AS builder
```

Comment:
also add `Dockerfile.dev`

Reply:
also `.env` file

Reply:
Require valid certs for oauth to google sheets, that's why dockerfile.dev is missing